### PR TITLE
Made common widgets for input fields and form labels, made utilities …

### PR DESCRIPTION
**Pull request by MattPutnam**
…for key listener filters and clearing form elements.

Style and simple functions for styling components put in new style.cljs, custom React components put in components.cljs.

### CODE REVIEW

File: src/cljs/org/broadinstitute/firecloud_ui/common/style.cljs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+(def colors {:link-blue "#6690c5"
+             :line-gray "#e6e6e6"
+             :button-blue "#457fd2"
+             :background-gray "#f4f4f4"
+             :text-light "#7f7f7f"
+             :text-gray "#666"})
+
+
+(defn form-label [text]
```
Comment (dmohs) [inline src/cljs/org/broadinstitute/firecloud_ui/common/style.cljs]: Is it too verbose to do `create-form-label`, `create-hint`, etc. to distinguish these from other things like `(def input-text-style ...)`, etc.?

File: src/cljs/org/broadinstitute/firecloud_ui/main.cljs [hunk 2]
```diff
@@ -1,11 +1,11 @@
 (ns org.broadinstitute.firecloud-ui.main
   (:require
    [dmohs.react :as react]
-   [org.broadinstitute.firecloud-ui.common :as common]
    [org.broadinstitute.firecloud-ui.page.method-repo :as method-repo]
    [org.broadinstitute.firecloud-ui.page.workspaces :as workspaces]
    [org.broadinstitute.firecloud-ui.session :as session]
    [org.broadinstitute.firecloud-ui.utils :as utils]
+   [org.broadinstitute.firecloud-ui.common.style :as style]
```
Comment (dmohs) [inline src/cljs/org/broadinstitute/firecloud_ui/main.cljs]: Nit: alphabetical

File: src/cljs/org/broadinstitute/firecloud_ui/page/method_repo.cljs [hunk 3]
```diff
@@ -3,7 +3,9 @@
    [clojure.string]
    [dmohs.react :as react]
    [org.broadinstitute.firecloud-ui.common :as common]
+   [org.broadinstitute.firecloud-ui.common.style :as style]
    [org.broadinstitute.firecloud-ui.common.table :as table]
+   [org.broadinstitute.firecloud-ui.common.components :as comps]
```
Comment (dmohs) [inline src/cljs/org/broadinstitute/firecloud_ui/page/method_repo.cljs]: Nit: alphabetical

**Comment by dmohs** [comment]
Really nice refactoring. Love it. One question about naming. Thanks for doing this.

File: src/cljs/org/broadinstitute/firecloud_ui/common/style.cljs (hunk 1 above)
Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/common/style.cljs]: My idea was to visually match other types of elements which are just nouns.  I could make the style maps private.

**Pull request merged**